steps/PageSlugs.ts

```diff
@@ -11,6 +11,7 @@ enum PageSlugs {
   Outside = "outside",
   Start = "start",
   AboutVisit = "about-visit",
+  FirstAttempt = "first-attempt",
```

Comment:
Let's group the unable to enter proprty journey in its own region.

```suggestion

  // Unable to enter
  FirstFailedAttempt = "first-failed-attempt",
```